examples/globe.js

```diff
@@ -9,18 +9,26 @@ const viewerDiv = document.getElementById('viewerDiv');
 
 // Instanciate iTowns GlobeView*
 const globeView = new itowns.GlobeView(viewerDiv, positionOnGlobe, { renderer });
+const menuGlobe = new GuiTools('menuDiv');
```

Comment:
oh ok, yeah, maybe a `this.view = view` should be added. Not worth the trouble here, we'll do it later.